app/lib/actions/katello/repository/update.rb

```diff
@@ -9,10 +9,10 @@ def plan(repository, repo_params)
           if (SETTINGS[:katello][:use_cp] && SETTINGS[:katello][:use_pulp]) && repository.library_instance?
             plan_action(::Actions::Candlepin::Product::ContentUpdate,
                         :content_id => repository.content_id,
-                        :name => repository.name,
+                        :name => repository.content.name,
```

Comment:
I think for custom repos the previous logic was correct as the content name comes from the repo name and if we rename it we want to change the name.  For redhat content we should probably just not include name and label and gpg_key_url in the hash.

In fact i bet the fact that gpg_key_url is being included will break things too.